projects/plugins/jetpack/modules/sharedaddy/sharing-service.php

```diff
@@ -71,20 +72,6 @@ public function get_all_services( $include_custom = true ) {
 			'skype'            => 'Share_Skype',
 		);
 
-		/**
-		 * Filters if Email Sharing is enabled.
-		 *
-		 * E-Mail sharing is often problematic due to spam concerns, so this filter enables it to be quickly and simply toggled.
-		 * @module sharedaddy
-		 *
-		 * @since 5.1.0
-		 *
-		 * @param bool $email Is e-mail sharing enabled? Default false if Akismet is not active or true if Akismet is active.
-		 */
-		if ( apply_filters( 'sharing_services_email', Jetpack::is_akismet_active() ) ) {
```

Comment:
Since we are getting rid of different hooks, I think we should add them all to the list of deprecated hooks here:
https://github.com/Automattic/jetpack/blob/ccc9d168ce12931360b795a2e6290afc0aef6420/projects/plugins/jetpack/class.jetpack.php#L5787-L5790

So folks will know to stop using them.

Reply:
Done. I've also removed the code that used these hooks, and marked the called functions as deprecated, as well as the `Jetpack_ReCaptcha` class. We might be able to remove those functions entirely, but that could cause trouble for users who may be using the functions and the `Jetpack_Recaptcha` class.